### PR TITLE
Adding maintainers to Redfish modules

### PR DESCRIPTION
**Pull request by jose-delarosa**
##### SUMMARY

Add more maintainers for new Redfish modules that were recently merged.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

.github/BOTMETA.yml

##### ANSIBLE VERSION

```paste below
ansible 2.7.0rc1 (detached HEAD 0202339461) last updated 2018/09/10 14:43:40 (GMT -500)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /share1/git/ansible/lib/ansible
  executable location = /share1/git/ansible/bin/ansible
  python version = 2.7.5 (default, May 31 2018, 09:41:32) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]

```

##### ADDITIONAL INFORMATION

I am author of modules. I am adding maintainers.

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -914,6 +914,7 @@ macros:
   team_openstack: emonty omgjlk juliakreger rcarrillocruz shrews thingee dagnello
   team_openswitch: Qalthos gdpak
   team_rabbitmq: chrishoffman manuel-sousa hyperized
+  team_redfish: jose-delarosa mraineri tomasg2012 billdodd
```
Comment (webknjaz) [inline .github/BOTMETA.yml:917]: Do @mraineri @tomasg2012 @billdodd know about this? Let's wait for their confirmation. Adding them might mean notification storm from bot pings.

@jose-delarosa please link to places where they've agreed to this or invite them to post that in this PR.

Comment (jose-delarosa) [reply .github/BOTMETA.yml:917]: Yes they know, we have an email thread. @mraineri @tomasg2012 @billdodd, can you please confirm here that you agree to be maintainers? Thank you.

Comment (webknjaz) [reply .github/BOTMETA.yml:917]: Okay, merging, then. It's better to keep such things public anyway.

**Comment by billdodd** [comment]
Yes, I agree to be a maintainer. I'm sure I'll have to set up some email
filtering for the notification storm. :-)

Bill

**Comment by jose-delarosa** [comment]
@webknjaz Favor, https://github.com/ansible/ansible/pull/44903 reverted this PR, I can't figure out why. Not a big deal I can resubmit another PR but did we not follow correct process the first time? Thanks.

**Comment by webknjaz** [comment]
Probably, b/c of incorrectly resolved merge conflict. Feel free to re-submit changes.

cc @gundalow

**Comment by gundalow** [comment]
`BOTMETA` restored in https://github.com/ansible/ansible/pull/45909

**Pull request merged**